src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.scss

```diff
@@ -2,7 +2,7 @@
 
 :host {
     display: block;
-    width: 500px;
+    width: 600px;
```

Comment:
Why not change `px` for `rems` from now on?

Reply:
True this, fotgot about it.